packages/sdks/javascript/src/api/generator/templates/apiInner.mustache

```diff
@@ -33,6 +33,9 @@ export class {{classname}}Service extends ApiBaseService {
      * @param {{=<% %>=}}{<%&dataType%>}<%={{ }}=%> {{^required}}[{{/required}}{{paramName}}{{^required}}]{{/required}} {{description}}
         {{/isHeaderParam}}
      {{/allParams}}
+     {{#isDeprecated}}
+     * @deprecated
+     {{/isDeprecated}}
```

Comment:
I believe `@deprecated` needs to be a highlighted to get he attention of somebody reading the code.
Could you please move it to become the very first JSDoc tag, before `@summary`?